polaris-core/src/test/java/io/polaris/service/storage/azure/AzureCredentialStorageIntegrationTest.java

```diff
@@ -250,10 +243,10 @@ public void testGetSubscopedTokenRead(boolean allowListAction, String service) {
   }
 
   @TestWithAzureArgs
-  public void testGetSubscopedTokenWrite(boolean allowListAction, String service) {
-    if (checkEnvNullVariables()) {
-      return;
-    }
+  public void testGetSubscopedTokenWrite(
+      @SuppressWarnings("unused") boolean allowListAction, String service) {
```

Comment:
why keep the parameter if we don't need it?

Reply:
it's a parameterized test